BankexWallet/InterfaceElements/DesignElements/SendTokenViewController/CustomSegues.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  CustomSegues.swift
+//  BankexWallet
+//
+//  Created by Георгий Фесенко on 28.05.2018.
+//  Copyright © 2018 Alexander Vlasov. All rights reserved.
+//
+
+import UIKit
+
+class ShowFavoritesSegue: UIStoryboardSegue {
+    override func perform() {
+        transition()
+    }
+    
+    func transition() {
```

Comment:
Too many magic numbers in this method. You should either put them into constants, or get them from view itself